models/Github/GithubManager.js

```diff
@@ -140,7 +140,10 @@ class GithubManager {
     }
 
     for (var channel of discordSubsystem.getPrimaryGuild().channels.array()) {
-      if (channel.id == config.discord_github_channel) {
+      if (channel.id == config.discord_github_channel && payload.sender.login == "yogstation13-bot") {
```

Comment:
uhhm. what?

Reply:
Making it so only yogbot PRs get posted in #botspam while all other PRs (IE PRs made by hoomans, not mirrorbots are posted in #coder-public

Reply:
You could also check if the PR's title starts with [MIRROR]

Reply:
We don't have any other bots though...